.git-blame-ignore-revs

```diff
@@ -0,0 +1,451 @@
+# This file includes the list of commits to exclude from `git blame`.
```

Comment:
Well I wasn't expecting to see this file.  But I'm not opposed to it.  It looks like an interesting addition.  I see this [feature](https://github.com/git/git/blob/master/Documentation/RelNotes/2.23.0.txt#L83-L84) was added to Git on April 20, and I see that Github isn't actually using it yet, but that some local clients are using it.  I think it will definitely give a better look at the actual blame and changes, for which I am in support.